packages/twenty-front/src/modules/ui/layout/page/components/DefaultLayout.tsx

```diff
@@ -69,6 +70,7 @@ export const DefaultLayout = () => {
   const theme = useTheme();
   const windowsWidth = useScreenSize().width;
   const showAuthModal = useShowAuthModal();
+  const showPlayground = useShowPlayground();
```

Comment:
I think this is too specific to playgrounds to deserve to be in DefaultLayout. Maybe we should do something more generic like useShowFullScreen?